src/server/snapshot.cc

```diff
@@ -95,19 +91,18 @@ void SliceSnapshot::Start(bool stream_journal, const Cancellation* cll, Snapshot
 
   VLOG(1) << "DbSaver::Start - saving entries with version less than " << snapshot_version_;
 
-  snapshot_fb_ = fb2::Fiber("snapshot", [this, stream_journal, cll] {
-    IterateBucketsFb(cll, stream_journal);
+  snapshot_fb_ = fb2::Fiber("snapshot", [this, stream_journal] {
+    this->IterateBucketsFb(stream_journal);
```

Comment:
Why the `this->` prefix?

Reply:
We are using this-> inside the lambda 10 lines below in another fibers lambda. So I decided to use the same syntax here :smile:

Reply:
It is not important for me, can revert